modules/meeting/app/controllers/recurring_meetings_controller.rb

```diff
@@ -27,9 +27,28 @@ def new
   end
 
   def show
-    respond_to do |format|
-      format.html
+    @meetings = collect_meetings
+  end
+
+  def collect_meetings
```

Comment:
@oliverguenther This needs a refactor, so don't focus on the details, but is there another way to do something like this? Merging the instantiated and non-instantiated meetings?